patch_parser_test.py

```diff
@@ -0,0 +1,18 @@
+import unittest
+from patch_parser import parse_comments
+from archive_converter import generate_email_from_file
+
+class Patch_Parser_Test(unittest.TestCase):
+
+    def test_parse_comments_for_single_email_thread(self):
+        email = generate_email_from_file("test_data/patch6.txt")
+        patchset = parse_comments(email)
+        self.assertTrue(len(patchset.patches) > 0)
+        self.assertTrue(patchset.patches[0].set_index == 0)
+        self.assertTrue(len(patchset.patches[0].text) > 0)
+        self.assertTrue(len(patchset.patches[0].text_with_headers) > 18)
```

Comment:
```suggestion
        self.assertGreater(len(patchset.patches[0].text_with_headers), 18)
```

This way it'll show the actual length.

Note: I didn't suggest this for all previous checks bc they were `> 0` and it'd be obvious what the length we got was.

Ditto these comments for the child commit.